rails_programming/introduction/installing_rails.md

```diff
@@ -63,27 +55,14 @@ If you don't get a version number drop by the chatrooms for some assistance.
 
 You won't need to use Yarn for this tutorial but you do need it installed for Rails to set up Webpack correctly.
 
-#### Step 1.3: Lay the Groundwork
-
-Next, if you haven't already done it, we need to create a directory that will house our project. You can name it anything you like!
-
-~~~bash
-mkdir odin_on_rails
-~~~
-
-Then, move into the new directory:
-
-~~~bash
-cd odin_on_rails
-~~~
-
-#### Step 1.4: Create the Application
+#### Step 1.3: Create the Application
 
 This is where things might start to be difficult to follow just what is happening. If you don't understand what's going on, just double check that you're typing in the correct commands and keep going. This section is meant to expose you to the process and to verify that everything is working. Again, it's OK to not understand what's going on at this point.
 
-We're going to start by telling Rails to initialize the application for us. Then, we'll tell Rails to create a template for us so that we can get up and running.
+We're going to start by navigating to the directory that you house your projects in, then telling Rails to initialize the application for us. Then we'll tell Rails to create a template for us so that we can get up and running.
```

Comment:
I find these instructions a little confusing because making a template is mentioned, which I believe is the scaffolding command later on, but the migrate database is not mentioned. However, in looking at the entire lesson, I think most of the time the instructions stick with the commands immediately following, so I'd remove this mention of creating the template.